### PR TITLE
Make changing quantity easier to click on

### PR DESCRIPTION
**Pull request by YKo20010**
#32

### CODE REVIEW

**Comment by cornell-appdev** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          6 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Skipping duplicate build file in Compile Sources build phase: /Users/vagrant/git/acai-ios/Models/MenuItem.swift (in target ‘acai-ios’)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Skipping duplicate build file in Compile Sources build phase: /Users/vagrant/git/acai-ios/UIFont+Shared.swift (in target ‘acai-ios’)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Skipping duplicate build file in Compile Sources build phase: /Users/vagrant/git/acai-ios/OrderDetailViewController.swift (in target ‘acai-ios’)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">SwiftLint not installed, download from https://github.com/realm/SwiftLint</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/cuappdev/acai-ios/blob/e18e7ee3fac0010ad44c48be3720848887752875/acai-ios/MenuViewController.swift#L59">acai-ios/MenuViewController.swift#L59</a> - TODOs should be resolved (coming back from detail view i…).<br />
<code>todo</code> <code>MenuViewController.swift:59</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/cuappdev/acai-ios/blob/e18e7ee3fac0010ad44c48be3720848887752875/acai-ios/MenuViewController.swift#L77">acai-ios/MenuViewController.swift#L77</a> - TODOs should be resolved (get menu from endpoint).<br />
<code>todo</code> <code>MenuViewController.swift:77</code></td>
    </tr>
  </tbody>
</table>

### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
MenuViewController.swift | 59 | TODOs should be resolved (coming back from detail view i...). (todo)
MenuViewController.swift | 77 | TODOs should be resolved (get menu from endpoint). (todo)


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: acai-ios/CollectionViewCells/QuantitySelectionCollectionViewCell.swift [hunk 1]
```diff
@@ -66,8 +66,8 @@ class QuantitySelectionCollectionViewCell: UICollectionViewCell {
     
     private func setupConstraints() {
         let borderWidth = 217
-        let buttonHeightWidth = 14
-        let buttonOffsetFromLabel = 17
+        let buttonHeightWidth = 40
+        let buttonOffsetFromLabel = 8
```
Comment (kevinchan159) [inline acai-ios/CollectionViewCells/QuantitySelectionCollectionViewCell.swift:70]: can we alphabetize these

**Review by kevinchan159 — APPROVED** [review]
looks great!

**Review by doctor3w — APPROVED** [review]
LGTM!

**Pull request merged**